spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/properties/ZeebeClientConfigurationProperties.java

```diff
@@ -408,7 +408,7 @@ public Duration getDefaultMessageTimeToLive() {
   @DeprecatedConfigurationProperty(
       replacement = "camunda.client.zeebe.base-url",
       reason = "plaintext is determined by the url protocol (http/https) now")
-  public boolean isPlaintextConnectionEnabled() {
+  public Boolean isPlaintextConnectionEnabled() {
```

Comment:
Changing "isPlaintextConnectionEnabled" return type to support null
value indicating that there is no configuration for it and avoiding
using default value when the value is actually not set.